classes/Configuration.php

```diff
@@ -254,6 +254,14 @@ public static function getGlobalValue($key, $idLang = null)
         return Configuration::get($key, $idLang, 0, 0);
     }
 
+    /**
+     * @deprecated use Configuration::getConfigInMultipleLangs() instead.
```

Comment:
```suggestion
     * @deprecated use Configuration::getConfigInMultipleLangs() instead
```
PHP CS Fixer wants this